packages/node/examples/hapi/index.js

```diff
@@ -23,12 +22,10 @@ const init = async () => {
       label: 'Owlbert',
       // User's email address
       email: 'owlbert@example.com',
-      startedDateTime: new Date(info?.received),
-      responseEndDateTime: new Date(info?.responded),
-      responseBody: response?.source,
     };
 
-    readmeio.log(process.env.README_API_KEY, req, res, payloadData, { fireAndForget: true });
+    req.body = request.payload;
```

Comment:
These changes should eventually be rolled into core so the framework specific implementation is as simple as possible.

Reply:
https://github.com/readmeio/metrics-sdks/issues/561